Makefile

```diff
@@ -2,7 +2,8 @@ DESTDIR   ?=
 PREFIX    ?= /usr/local
 MANPREFIX ?= $(PREFIX)/man
 CFLAGS += -Wall -Wextra
-LDLIBS += `pkg-config --libs ncurses 2>/dev/null || echo '-lcurses -ltinfo'`
+CFLAGS += `pkg-config --cflags ncursesw 2>/dev/null || echo '-D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600 -I/usr/include/ncursesw'`
+LDLIBS += `pkg-config --libs ncursesw 2>/dev/null || echo '-lcursesw -ltinfo'`
```

Comment:
@edgar-bonet I believe the `-ltinfo` may need to be `-ltinfow` with `w` also bease on:

```
# pkgconf --libs ncursesw
-lncursesw -ltinfow
```

Commit 5bdee13933ba65cef945f19ab1a3c998140b9feb compiles well on Gentoo :+1: 

Yes, it's two different files with regard to content:

```
# ls -l /usr/include/{ncursesw/,}{,n}curses.h
-rw-r--r-- 1 root root  82280 Apr 13  2023 /usr/include/curses.h
lrwxrwxrwx 1 root root      8 Apr 13  2023 /usr/include/ncurses.h -> curses.h
-rw-r--r-- 1 root root 100387 Apr 13  2023 /usr/include/ncursesw/curses.h
lrwxrwxrwx 1 root root      8 Apr 13  2023 /usr/include/ncursesw/ncurses.h -> curses.h
```

Reply:
I see. It's `-ltinfo` on Ubuntu and `-ltinfow` on Gentoo. Should I change the fallback to `-ltinfow`? If, as you suggest in a further comment, we drop the `pkg-config` fallbacks (presumably in another PR), then I can leave this as-is for now.

PS: thanks for testing and for the info on Gentoo's headers.